spec/rules_factory_spec.rb

```diff
@@ -0,0 +1,40 @@
+require "rspec"
+require "./rules_factory.rb"
+require "./item.rb"
+
+describe RulesFactory do
+  before(:each) do
+    @rule_factory = RulesFactory.new
+  end
+
+  it "found AgedBrie Rule" do
```

Comment:
don't use past-tense for names of your tests. I would personally probably just name this "aged brie".